### DISCUSSION TITLE
Adding "Quick Load" to the game

### DISCUSSION THREAD

**Discussion by agergo100** [Wohlstand/TheXTech#489] [Ideas]
A faster way to reload your save instead of entering a level, hit pause and "**Exit**" if you don't want to hit "**Save and Exit**".
Very useful if you want quickly restore your lost items when you died.

**Comment by Alucard648** [comment]
Oh yes. It took 8 attempts to beat Princess Cliche final boss as Hammer Mario. Having to do that thing after every fail is annoying.

**Comment by Wohlstand** [comment]
And this thing shouldn't be available when speedrunning mode 2 and higher is active to ensure the fairness of vanilla episodes speedrunning (this feature gives a serious time winning, for casual playthrough it's okay, but for speedrunning mode, it's a significant environment change).
